wsproto/connection.py

```diff
@@ -159,3 +161,35 @@ def events(self):
                     )
         except ParseFailed as exc:
             yield CloseConnection(code=exc.code, reason=str(exc))
+
+
+class H2Wrapper(object):
```

Comment:
I'm not sure if anyone is actually going to use this wrapper in production.
Maybe this would be better moved to an example script?

Reply:
I agree with you, I'll move

Reply:
Sorry I'm wrong, this is required if you use the H2Handshake.

Reply:
Maybe move it into the H2Handshake then?
I have no strong opinion either way.